pages/ksphere/kommander/1.2/release-notes/index.md

```diff
@@ -10,75 +10,55 @@ enterprise: false
 
 # Supported Versions
-
 | Kubernetes Support | Version |
 | ------------------ | ------- |
 | **Minimum**        | 1.16.0  |
-| **Maximum**        | 1.19.x  |
-| **Default**        | 1.18.10  |
-
-# Kommander 1.2.0 Beta 1 (September 14th, 2020)
-
-## Features/Improvements
-
-- Improved catalog handling
-- Improved Access Control for editing/deleting actions
-- Added a global Infrastructure Provider view
-
-## Bug Fixes
-
-- Fixed an issue where logout wasn't deleting any cookies
-- Smaller UX Bugs and Improvements
-
-## Component Versions
-
-- Addon: `1.2.0-9`
-- Chart: `0.11.3`
-- auto-provisioning (yakcl): `0.4.5`
-- kommaner-federation (yakcl): `0.4.5`
-- kommander-licensing (yakcl): `0.4.5`
-- UI: `6.22.0`
-- kommander-karma: `0.3.10`
-- kubeaddons-catalog: `0.1.12`
-- kommander-thanos: `0.1.15`
-- kubecost: `0.1.12`
-- grafana: `4.6.3`
-
-# Kommander 1.2.0 Beta 0 (August 27th, 2020)
-
-## Features/Improvements
-
+| **Maximum**        | 1.18.x  |
+| **Default**        | 1.18.8  |
+
+# New Features and Capabilities 
+
+## Air gapped environments
+Kommander now supports installing and running in an air gapped environment, either on premises or in the cloud. Air gapping is ideal for instances where high-security is a must or where Internet connectivity is either undesirable or unavailable. For more details on setting up Kommander in an air gapped environment, see [Install Kommander air gapped][install_airgapped].
+
+## Component versions
+- Addon: 1.2.0-26
+- Chart: 0.12.3
+- kommander-federation (yakcl): 0.6.2
+- kommander-licensing (yakcl): 0.6.2
+- UI: 6.37.0
+- kommander-karma: 0.3.10
+- kubeaddons-catalog: 0.1.15
+- kommander-thanos: 0.1.15
+- kubecost: 0.1.15
+- grafana: 4.6.3
+
+## Fixed and Improved Issues 
+- Improved Access Control for editing/deleting actions. 
 - Changed routing from hash-based to history-based to allow redirects after login.
-
-## Bug Fixes
-
-- Smaller UX Bugs and Improvements
-
-## Component Versions
-
-- Addon: `1.2.0-5`
-- Chart: `0.10.3`
-- auto-provisioning (yakcl): `0.4.2`
-- kommaner-federation (yakcl): `0.4.2`
-- kommander-licensing (yakcl): `0.4.2`
-- UI: `6.22.0`
-- kommander-karma: `0.3.10`
-- kubeaddons-catalog: `0.1.12`
-- kommander-thanos: `0.1.15`
-- kubecost: `0.1.12`
-- grafana: `4.6.3`
+- Added SAML IDP.
+- Moved provisioning code to Konvoy, Kommander now uses Konvoy to provision clusters.
+- Added a setting to configure thresholds for resource warnings. 
+- Added the ability to show metrics in attached EKS clusters.
+- Added Infrastructure provider management on global level.
+- Fixed an issue where logout was not deleting any cookies.
+- Fixed UI crashes when kubecost was down.
+- Fixed catalog API occasionaly returned 500.
+- Fixed deleting cluster labels was impossible.
+- Fixed counting self-attached host cluster against license count.
```

Comment:
This is very unclear, and probably important.  Does the following capture the issue?

```suggestion
- Fixed an issue that erroneously counted the self-attached host cluster against the license count.
```

Reply:
@cmastr @russurquhart1 you good with that? ☝️